tuned/plugins/plugin_cpu.py

```diff
@@ -408,6 +427,46 @@
 		except (ValueError, TypeError):
 			return None
 
+	def _get_uncore_data(self):
+		out, files = dict(), ['initial_max_freq_khz', 'max_freq_khz']
+		dir = '/sys/devices/system/cpu/intel_uncore_frequency'
+		for dirname, dirnames, filenames in os.walk(dir):
+			pkg, ignore = dict(), False
+			for file in files:
+				if file not in filenames:
+					ignore = True
+					break
+				fpath = os.path.join(dirname, file)
+				data = self._cmd.read_file(fpath, err_ret=None, no_error=True)
+				try:
+					value = int(data.strip())
+				except:
```

Comment:
Will fix in the next version, once issue with the general direction regarding profile is clear.

Reply:
So I assume from this that the "int" call has its own baseexception that I'm overriding. Not sure what the problem is with that but I can replace this with a simple "if re.match('[0-9]+', data.strip()):". That try/catch is just there in case the sysfs file returns garbage (e.g. anything other than a decimal integer). If it happens I ignore its package and move on like it doesn't exist.